docs/docsource/Installation/upgrade/upgrading_zenko.rst

```diff
@@ -11,11 +11,16 @@ To upgrade Zenko:
 
 #. Unpack the .zip or .tar.gz file and navigate to Zenko/kubernetes/. 
 
-#. Enter this Helm commamnd, inserting your Zenko server's name
+#. Enter this Helm commamnd, inserting your Zenko server's release name
    :: 
 
       $ helm upgrade {{zenko-server-name}} ./zenko
 
+   If you are used a custom values be sure to reuse this file on upgrades.
```

Comment:
"used a custom values" -> "using custom values"

Reply:
Both fixed. Thank you!